modules/exploits/multi/http/joomla_http_header_rce.rb

```diff
@@ -5,7 +5,7 @@
 
 require 'msf/core'
 
-class Metasploit3 < Msf::Exploit::Remote
+class Metasploit < Msf::Exploit::Remote
```

Comment:
I'm assuming you are just changing this one as an example and if all goes well someone will update all other modules?